src/docfx/template/liquid/JavaScriptTag.cs

```diff
@@ -10,7 +10,8 @@ internal class JavaScriptTag : Tag
     {
         public override void Render(DotLiquid.Context context, TextWriter result)
         {
-            result.Write($@"<script src=""{LiquidTemplate.GetThemeRelativePath(Markup)}"" ></script>");
+            var templateBasePath = (string?)context.Registers["template_base_path"];
+            result.Write($@"<script src=""{LiquidTemplate.GetThemeRelativePath(templateBasePath, Markup)}"" ></script>");
```

Comment:
consider pass `context` to `GetThemeRelativePath` method to reuse the template base path retrieval logic?
```suggestion
            result.Write($@"<script src=""{LiquidTemplate.GetThemeRelativePath(context, Markup)}"" ></script>");
```